runtime/dolphin/src/weights/frame_system.rs

```diff
@@ -0,0 +1,152 @@
+// Copyright 2020-2022 Manta Network.
```

Comment:
You need a CI workflow like this https://github.com/Manta-Network/Manta/blob/manta/.github/workflows/generate_calamari_weights_files.yml in order to benchmark whatever pallets you need appropriately. Same AWS bare-metal instance and environment. We shouldn't reuse Calamari's weights because the runtime configurations will differ which has an effect on the weights calculation.

Reply:
We don't need to do this for now. (but need to log the issue)

Reply:
https://github.com/Manta-Network/Manta/issues/376